bokeh/models/widgets/tables.py

```diff
@@ -14,6 +14,8 @@
 import logging # isort:skip
 log = logging.getLogger(__name__)
 
+from Lib.typing import Sequence
```

Comment:
In this case `Sequence` is not from Python, it is one of the Bokeh  property types, so it should go in the list of all the property imports below.

Reply:
Under `core.properties` on line 229 I am finding is `Seq` listed under `__all__ = (` , I didn't see one in the `core.properties.py` named `Sequence`. I am probably just overlooking it though? `Seq` looks like it is a `ContainerProperty`.

Reply:
@jensencl Apologies, `Seq` is actually what you want, I misspoke. That Bokeh property type will accept most any sequence type (lists, tuples, arrays, series, etc.)